### DISCUSSION TITLE
[Feature] Add to album from map view

### DISCUSSION THREAD

**Discussion by lividhen** [immich-app/immich#10411] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

An add to album button when an image is selected on the map view. This would make it easier to add specific photos from a location to an album.

The lack of an add to album button feels a little silly considering there is a favorite button:
![Screenshot_20240616_161528](https://github.com/immich-app/immich/assets/42221797/a18db88a-778b-412b-bb90-cbac0b07dd3d)

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by deployHuman** [comment]
yes this! i would love to beable to add a whole cluster of images from the maps to an album.
I can open the cluster and one-by-one add them, but thats insane.
would be simple as "add all" in the bubble your are viewing or something
